lib/ansible/executor/action_write_locks.py

```diff
@@ -15,30 +15,44 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-# Make coding more python3-ish
-from __future__ import (absolute_import, division, print_function)
+from __future__ import absolute_import, division, print_function
```

Comment:
Nit: it's best not to mix up any formatting updates in the same PR as the functional changes. The title doesn't say "reformat things". The changes in the patch should only relate to what the title states (and description for that matter) — this is called an atomic change.

Reply:
I think my editor did this auto-magically, good call.